src/AutorestSwift/View Models/ParameterViewModel.swift

```diff
@@ -38,9 +38,9 @@ struct ParameterViewModel {
 
     init(from param: ParameterType, withName specificName: String? = nil) {
         if let name = specificName, !name.isEmpty {
-            self.name = name
+            self.name = name.firstLowerCased
         } else {
-            self.name = param.name
+            self.name = param.name.firstLowerCased
         }
```

Comment:
Can we not control this in our autorest.swift configuration? We eliminated most if not all casing logic in our code by using what autorest configuration already provides.

Reply:
Revert this change.